src/components/TutorialHeader/TutorialHeader.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import {
+  Header,
+  HeaderName,
+  HeaderNavigation,
+  HeaderMenuItem,
+  HeaderGlobalBar,
+  HeaderGlobalAction,
+  SkipToContent,
+} from 'carbon-components-react/lib/components/UIShell';
+import Notification20 from '@carbon/icons-react/lib/notification/20';
+import UserAvatar20 from '@carbon/icons-react/lib/user--avatar/20';
+import AppSwitcher20 from '@carbon/icons-react/lib/app-switcher/20';
+import { Link } from 'react-router-dom';
+
+const TutorialHeader = () => (
+  <Header aria-label="Carbon Tutorial">
+    <SkipToContent />
+    <HeaderName element={Link} to="/" prefix="IBM">
+      Carbon Tutorial
+    </HeaderName>
```

Comment:
You're missing the `<HeaderNavigation>` component. See https://www.carbondesignsystem.com/tutorial/react/step-1/#import-and-export-the-header